legal/security.md

```diff
@@ -60,6 +60,25 @@ However, to allow us to automatically specify SSH keys, service hook configurati
 
 We only manually access your code when explicitly requested by you and only in explicit consent with you, and only to debug and help solve build issues.
 
+### How does Travis access my Bitbucket account?
+When you sign up for Travis, we collect an OAuth token from Bitbucket, which allows us to request data from the Bitbucket API on your behalf. This OAuth token is stored securely in our database and is protected from unauthorized access.
+
+The token is bound to permissions set on Bitbucket, so please make sure you've read their documentation on access control and API access permissions.
+
+We use this token in these situations, and under no other circumstances than described below.
+
+- To synchronize the repositories you have access to. We use this information to show you the available repositories on your profile page so you can enable or disable building them on Travis.
```

Comment:
We have to add 'To  synchronize the teams...

Reply:
please add this information directly in the file in the branch